### ISSUE TITLE
Update Failed

### ISSUE THREAD

**Issue by Randomnamethatisnottakin** [dalaranwow/dalaran-wow#3926]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Started playing here last night, no issue. Today I get a update screen and failed restart on log in, Using provided client, correct realmlist and tried the repair tool which also fails. Can't get help on the forum, I can log in and make a post on the dalaran site, but the post goes to a ddos check then doesn't post at all. Character name : Nightcap -Kirin tor realm

**Expected behaviour**: The game should continue to work, and the site should allow me to post for help. I don't know where else to get help, since the site won't let me post when I'm logged into it.

**Steps to reproduce the problem**: Make a new account and try logging in I guess, I'm new here, not sure if this is a normal problem for this server.

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Make shure that all files in you wow folder have removed the write protection. Also please try again posting you issue in the forum. Sometimes the post will not be created if caps or special links are used.

Let me know if it helped you. I will search for futher solutions if not.

However, that's only the issue-tracker and it's only related to ingame/serverside issues. 

/closing (you can still answer in this ticket)

**Comment by Randomnamethatisnottakin** [comment]
Thanks for the reply, I made sure that all the files in my wow folder have write permissions and the problem has not changed. I will try the forum again, But I don't think it will work this time either, as I did not use caps or links in the last few times I tried to use it. 05.12.2016, 00:36, "Rushor" <notifications@github.com>:Make shure that all files in you wow folder have removed the write protection. Also please try again posting you issue in the forum. Sometimes the post will not be created if caps or special links are used.Let me know if it helped you. I will search for futher solutions if not.However, that's only the issue-tracker and it's only related to ingame/serverside issues./closing (you can still answer in this ticket)—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.   chris

**Comment by Rushor** [comment]
@ Randomnamethatisnottakin 

another problem that occurs sometimes:

You need to enable "run as administrator" for the wow.exe.

**Comment by Rushor** [comment]
please continue the discussion here http://www.dalaran-wow.com/forums/input-and-assistance/community-support/topic/9453/failed-update-can-t-play-anymore
